frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -153,26 +182,45 @@ const SearchField: React.FC = () => {
     setInputValue("");
   };
 
+  const handleOpen = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
   return (
-    <Container>
-      <Autocomplete
-        autoComplete
-        freeSolo
-        selectOnFocus
-        size="small"
-        inputValue={inputValue}
-        renderInput={Input}
-        renderOption={renderResult}
-        onInputChange={onInputChange}
-        open={showOptions}
-        onOpen={onOptionsOpen}
-        onClose={onOptionsClose}
-        options={options}
-        filterOptions={filterResults}
-        getOptionLabel={x => x.label}
-        ListboxProps={{ style: { backgroundColor: fade(theme.palette.secondary.main, 0.75) } }}
-      />
-    </Container>
+    <Grid container alignItems="center">
+      {open ? (
+        <Container>
+          <ClickAwayListener onClickAway={handleClose}>
+            <Autocomplete
+              autoComplete
+              freeSolo
+              selectOnFocus
+              size="small"
+              inputValue={inputValue}
+              renderInput={Input}
+              renderOption={renderResult}
+              onInputChange={onInputChange}
+              open={showOptions}
+              onOpen={onOptionsOpen}
+              onClose={onOptionsClose}
+              options={options}
+              filterOptions={filterResults}
+              getOptionLabel={x => x.label}
+              PopperComponent={renderPopper}
+              closeIcon={<CustomCloseIcon />}
+            />
+          </ClickAwayListener>
+        </Container>
+      ) : (
+        <SearchIconButton onClick={handleOpen}>
+          <FiSearch />
```

Comment:
imported from https://react-icons.github.io/search, better matches the one on figma

Reply:
okay will use mui icons instead